pages/favorites/index.js

```diff
@@ -0,0 +1,57 @@
+import VerticalGameList from "@/components/VerticalGameList";
+import styled from "styled-components";
+import useSWR from "swr";
+import { CiFolderOff } from "react-icons/ci";
+
+export default function FavoritesPage({
+  isFavorite,
+  toggleFavorite,
+  localGameData,
+}) {
+  const { data, isLoading, error } = useSWR("/api/games");
```

Comment:
this approach will work for now, but in the future it might be not useful to fetch all games and filter here in the frontend. Then you might need to create a new api endpoint and use the Game Model to find the games saved in the favorites Array directly.